internal/config.go

```diff
@@ -37,6 +37,8 @@ type IRCSettings struct {
 type TelegramSettings struct {
 	Token               string `env:"TELEIRC_TOKEN,required"`
 	ChatID              int64  `env:"TELEGRAM_CHAT_ID,required"`
+	Prefix              string `env:"TELEGRAM_MESSAGE_PREFIX" envDefault:"<"`
+	Suffix              string `env:"TELEGRAM_MESSAGE_SUFFIX" envDefault:"> "`
```

Comment:
So are these two settings going to be separate from the IRC settings now?

Reply:
yeah, these settings are what shows up on IRC from Telegram.